backend/layers/business/business.py

```diff
@@ -667,21 +708,35 @@ def publish_collection_version(self, version_id: CollectionVersionId) -> None:
             raise CollectionPublishException("Cannot publish a collection with datasets of different schema versions")
         schema_version = next(iter(schema_versions))
 
+        date_of_last_publish = datetime.min
```

Comment:
This could be the default value for `get_all_dataset_versions_for_collection(from_date=datetime.min)`

Reply:
That actually already is the default value. Slightly annoying I agree but the calling logic dictates that we pass the `date_of_last_publish` no matter what; in some cases it will be `None`, which overrides the default, so then we handle that case in the method itself.